README.md

```diff
@@ -7,6 +7,8 @@
 
 embedchain is a framework to easily create LLM powered bots over any dataset. If you want a javascript version, check out [embedchain-js](https://github.com/embedchain/embedchainjs)
 
+Chinese Colab Tutorial:https://colab.research.google.com/drive/10_7Y0x4YXWVjuhhYwVraGQLpKAatTQTm?usp=sharing
```

Comment:
Hey that's a nice initiative. thanks.
One request: can you please create a section called Colab Notebooks just before Advanced section
and then list yours there?

Reply:
Done